package.json

```diff
@@ -11,7 +11,8 @@
     "prepublish": "./node_modules/.bin/babel src --out-dir lib",
     "pretest": "./node_modules/.bin/babel src --out-dir lib",
     "test-run": "./node_modules/.bin/mocha --compilers js:babel-core/register",
-    "test": "npm run lint && npm run test-run"
+    "test": "npm run lint && npm run test-run",
+    "prepare": "npm run prepublish"
```

Comment:
Nice! @tshaddix if we mix this with Travis, can we get rid of having to create a specific release and just automate via CI/CD when something lands on `master`?

Reply:
@vhmth Absolutely. Would love to chat about this separately.

Reply:
https://github.com/tshaddix/react-chrome-redux/issues/137